code/modules/holiday/holidays.dm

```diff
@@ -640,3 +640,13 @@
 
 /datum/holiday/hebrew/passover/getStationPrefix()
 	return pick("Matzah", "Moses", "Red Sea")
+
+/datum/holiday/pride_week
+	name = "Pride Week"
+	begin_day = 1
+	begin_month = JUNE
+	end_day = 7
+	end_month = JUNE
+
+/datum/holiday/pride_week/getStationPrefix()
+	return pick("Trans Rights Are Human Rights","Stonewall","LGBTQIAP+","Justice For George Floyd","Black Lives Matter","Trans Rights","LGBTQIAP+ Rights", "Pride")
```

Comment:
a lot of these, while fitting slogans for pride week, are either way too long as station prefixes or prefixes that don't make sense in 2560

Reply:
also having LGBTQIAP+ twice is an odd choice

Reply:
one is just the acronym, one is the acronym and rights, it's fine